.gitignore

```diff
@@ -1 +1 @@
-src/__pycache__
+src/is_face_detector/__pycache__
```

Comment:
your gitnignore file is specific for your repository. Try to use a more generic approach like: https://github.com/github/gitignore/blob/master/Python.gitignore